### ISSUE TITLE
Read compressed files directly.

### ISSUE THREAD

**Issue by justinmcgrath** [CloudCompare/CloudCompare#1256]
**Describe the feature you would like**
Three column text files of point clouds compress well, so it is nice to store them compressed. However, then they need to be decompressed to open in CloudCompare. It would be nice if CloudCompare recognized common compression types and decompressed automatically. That is, when dragging a compressed text file onto CloudCompare, it would open the same dialog as when dragging an uncompressed text file. Very common types are zip and gz, so those would be great. lzo and lz4 seem attractive since although they have slightly worse compression ratio than gz, they decompress incredibly quickly. 

**Describe alternatives you've considered**
This is not necessary and one can decompress the files first or not compress them. Based on experience with other programs that handle compressed files automatically, it is quite convenient though.

**Is your feature request related to a problem? Please describe.**
Our group has 50 GB of point cloud data, and we're adding more. With COVID restrictions we're all working at home, and it is inconvenient to transfer these files. With compression the size could be less than half, and decompression with all formats I looked at is fast. For lz4 it's very fast.

**Additional context**
Testing this with about 150 megabytes in about 100 files, I get these results, compressing files individually, reporting the total size and times for all files:

|program|compressed size| compression time | decompression time|
|------|----------|-----------|-------|
|gzip | 40 MB |  6.15 s | 0.7 s|
|lzo -7 | 55 MB | 13 s | 0.34 s|
|lz4 -7 | 56 MB | 5.8 s | 0.13 s|

**Comment by dgirardeau** [comment]
Silly question, but have you considered another (smarter) format than ASCII/text file? Cause any standard and 'open' binary format such as PLY or E57 will already save a lot of space. And the compressed LAZ format will save even more space.

**Comment by dgirardeau** [comment]
(oops)

**Comment by justinmcgrath** [comment]
Well, I had not considered that. The group is mostly scientists and although we are decent regarding statistics and algorithm development, a lot of the computer science parts we're not familiar with. It appears that there are modules for LAS and PLY in Python, so I will try those. I assume you're suggesting them because CloudCompare can open them directly?

**Comment by dgirardeau** [comment]
Yes. If you can find a module to load LAZ explicitly, it will be even more efficient.
